hack/check_pre_req.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/env bash
+
+# Copyright 2024 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e # exit on error
+
+echoerr() {
+    echo "$@" 1>&2;
+}
+
+is_installed() {
+    # $1 == name
+    # $2 == command name to search
+    # $3 == command to get the evrsion
+    # $4 == help
+    if which $2 > /dev/null ; then
+        echo -e "\u2705 $1"
+        if [ $# -gt 2 ]; then
+            echov "   version: $(eval "$3" 2> /dev/null)"
+        fi
+        echov "      path: $(which $2)"
+    else
+        echo -e "\u274c $1"
+        if [ $# -gt 3 ]; then
+            echov "   install: $4"
```

Comment:
Maybe make it a little more explicit: say "how to install" rather than just "install"?

Reply:
ok, thank you